### PR TITLE
OCA discord server

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Comment by SimoRubi** [comment]
Thanks! Is that a permanent invite link?

**Comment by eLBati** [comment]
It is

**Review by SimoRubi — APPROVED** [review]
> > Thanks! Is that a permanent invite link?
>
> It is

Then it's ok for me 😃

**Review by marcelsavegnago — APPROVED** [review]

File: website/Contribution/CONTRIBUTING.rst [hunk 1]
```diff
@@ -1160,8 +1160,7 @@ Please respect a few basic rules:
   `OCA Core Maintainers <https://github.com/orgs/OCA/teams/core-maintainers>`_.
   can do the job. You can notify them on Github using ``@OCA/core-maintainers``)
 * If you are in a hurry just send a mail at
-  contributors@odoo-community.org or ask by IRC (FreeNode
-  oca, openobject channel).
+  contributors@odoo-community.org or ask by chat (`OCA discord server <https://discord.gg/rN5xRdE>`_).
```
Comment (Yajo) [inline website/Contribution/CONTRIBUTING.rst:1163]: It'd be nice to mention how it's bridged to Matrix community [+oca:matrix.org](https://matrix.to/#/+oca:matrix.org)

**Comment by eLBati** [comment]
> A very small description would clarify a lot IMHO

@luisg123v a description of?

File: website/Contribution/CONTRIBUTING.rst (hunk 1 above)
Comment (eLBati) [reply website/Contribution/CONTRIBUTING.rst:1163]: How is it bridged to Matrix?

Comment (Yajo) [reply website/Contribution/CONTRIBUTING.rst:1163]: Check the announcement in https://odoo-community.org/groups/contributors-15/contributors-183472?mode=thread&date_begin=&date_end=

**Comment by luisg123v** [comment]
@eLBati I mean a commit description, because it's empty.

**Comment by eLBati** [comment]
@luisg123v e0af206a468c0c62c64d1faf958951b3b4063450 "OCA discord server" :thinking:

File: website/Contribution/CONTRIBUTING.rst (hunk 1 above)
Comment (eLBati) [reply website/Contribution/CONTRIBUTING.rst:1163]: @Yajo What do you suggest to write? Thanks

Comment (Yajo) [reply website/Contribution/CONTRIBUTING.rst:1163]:
```suggestion
  contributors@odoo-community.org or ask by chat in either of:
  
  - `OCA discord server <https://discord.gg/rN5xRdE>`_
  - `Matrix community +oca:matrix.org <https://matrix.to/#/+oca:matrix.org>`__ (bridged to discord).
```

Comment (eLBati) [reply website/Contribution/CONTRIBUTING.rst]: Done, thanks

**Review by Yajo — APPROVED** [review]

**Review by dreispt — APPROVED** [review]

**Comment by dreispt** [comment]
@OCA/board

**Pull request merged**